keps/sig-windows/1001-windows-cri-containerd/README.md

```diff
@@ -100,40 +100,49 @@ Additionally, users could choose to run with only CRI-ContainerD instead of Dock
 
 ## Proposal
 
 ### User Stories
 
-#### Improving Kubernetes integration for Windows Server containers
+#### Increasing the parity between Kubernetes on Linux and Windows Server containers
 
 Moving to the new Windows HCSv2 platform and ContainerD would allow Kubernetes to add support for:
 
 - Mounting single files, not just folders, into containers
 - Termination messages (depends on single file mounts)
 - /etc/hosts (c:\windows\system32\drivers\etc\hosts) file mapping
 
-#### Improved isolation and compatibility between Windows pods using Hyper-V 
+#### Increasing the forward-compatibility and security of Windows pods by leveraging Hyper-V
 
-Hyper-V enables each pod to run within it’s own hypervisor partition, with a separate kernel. This means that we can build forward-compatibility for containers across Windows OS versions - for example a container built using Windows Server 1809, could be run on a node running Windows Server 1903. This pod would use the Windows Server 1809 kernel to preserve full compatibility, and other pods could run using either a shared kernel with the node, or their own isolated Windows Server 1903 kernels. Containers requiring 1809 and 1903 (or later) cannot be mixed in the same pod, they must be deployed in separate pods so the matching kernel may be used. Running Windows Server version 1903 containers on a Windows Server 2019/1809 host will not work.
+Currently, the only way to run Hyper-V isolation in Kubernetes is to use the deprecated `experimental` isolation type, which was added in v1.10.
+
+Hyper-V is capable of running a pod within it’s own hypervisor partition, with a separate kernel.  This means that we can build forward-compatibility for containers across Windows OS versions - for example a container built using Windows Server 1809, could be run on a node running Windows Server 1903. This pod would use the Windows Server 1809 kernel to preserve full compatibility, and other pods could run using either a shared kernel with the node, or their own isolated Windows Server 1903 kernels. Containers requiring 1809 and 1903 (or later) cannot be mixed in the same pod, they must be deployed in separate pods so the matching kernel may be used. Running Windows Server version 1903 containers on a Windows Server 2019/1809 host will not work.
 
 In addition, some customers may desire hypervisor-based isolation as an additional line of defense against a container break-out attack.
 
-Adding Hyper-V support would use [RuntimeClass](https://kubernetes.io/docs/concepts/containers/runtime-class/#runtime-class).
+We propose adding scheduling support directly into Kubernetes by leveraging the [RuntimeClass](https://kubernetes.io/docs/concepts/containers/runtime-class/#runtime-class) feature.
```

Comment:
This makes it sound like additional changes are needed to RuntimeClasses for Hyper-V isolated container support which is not the case.
Can we re-word this to something like 'Scheduling Hyper-V isolation containers we work by leveraging [RuntimeClass] functionality which already exists in Kubernetes today'?